libraries/package_deps.rb

```diff
@@ -0,0 +1,21 @@
+def jruby_package_deps
+  case node['platform_family']
+  when 'debian'
+    %w(make g++)
+  when 'freebsd'
+    %w( alsa-lib bash dejavu expat fixesproto fontconfig freetype2 gettext-runtime giflib indexinfo inputproto java-zoneinfo javavmwrapper kbproto libICE libSM libX11 libXau libXdmcp libXext libXfixes libXi libXrender libXt libXtst libfontenc libpthread-stubs libxcb libxml2 mkfontdir mkfontscale openjdk8 recordproto renderproto xextproto xproto )
```

Comment:
hell knows. Honestly I'd love to go through this package list at some point and slim it down. 

I'll leave this in TODO: for now.